app/controllers/comments_controller.rb

```diff
@@ -17,11 +17,12 @@ def create
 
     respond_to do |format|
       if @comment.save
-        format.html { redirect_to @post, notice: 'Comment was successfully created.' } # changed the redirect to @post
+        format.html { redirect_to @post, notice: 'Comment was successfully created.' }
 
-        def comment_params
-          params.require(:comment).permit(:author => :current_user, :post_id, :text)
-        end
+        comment_params
+        params.require(:comment)
+        params.permit(author: :current_user)
       end
```

Comment:
- [ ] I applaud you for making use of `params method`. You can read more on [strong parameters](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)

Reply:
@t-yanick Thanks for your support. About the files, they show for me. Not sure what's going on.